regression/ansi-c/static4/main.c

```diff
@@ -0,0 +1 @@
+static int foo(int a);
```

Comment:
This would still pass with the old logic since it is both static and without body. The function should have a body and should still be excluded.